config.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from azure.identity import DefaultAzureCredential
+
+# azure settings
+BLOB_STORE = {
```

Comment:
Normal casing can be used here. BLOB_STORE is too loud even if the variable contains a global constant. `config.cloud.blob_store` would be the ideal reference.

Reply:
Here `cloud` is what type of data structure, is it a class?